### DISCUSSION TITLE
Parallel Plotting?!

### DISCUSSION THREAD

**Discussion by SackofFluor** [Chia-Network/chia-blockchain#3072] [Q&A]
Hey Guys,

maybe there is somewhere some info about it, but I cant find it...

Does it make sense to plot as much as possible in parallel (as much as the temp-drive allow) or  how do you determine how much you plot in parallel?

**Comment by myxtype** [comment]
According to the number of CPU threads, SSD capacity, memory capacity to determine the number of parallel drawing.

It is best to draw a graph with two threads. Each k32 graph needs about 250GB of temporary SSD capacity, and the memory is at least 2000m.
